sdk/openai/Azure.AI.OpenAI/tests/OpenAIImageTests.cs

```diff
@@ -50,6 +50,16 @@ public async Task CanGenerateImages(Service serviceTarget)
             ImageGenerationData firstImageLocation = imageGenerations.Data[0];
             Assert.That(firstImageLocation, Is.Not.Null);
             Assert.That(firstImageLocation.Url, Is.Not.Null.Or.Empty);
+
+            // ContentFilterResults and PromptFilterResults are currently Azure Only
+            if (serviceTarget is Service.Azure)
```

Comment:
If `Service.Azure` is enum type, then it should be 

```suggestion
            if (serviceTarget == Service.Azure)
```

Reply:
Resolved by https://github.com/ninghu/azure-sdk-for-net/pull/16/commits/90f2f2c8fd894dbd52b52a0d34e947a046726edd